### PR TITLE
Add Pankaj as committer

### PR DESCRIPTION
**Pull request by kaxil**
Announcement thread: https://lists.apache.org/thread/xv8rphsmn0bw6y25ohdtyvxwwx319khp

### CODE REVIEW

File: landing-pages/site/content/en/announcements/_index.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ menu:
 
 **Note:** Follow [@ApacheAirflow](https://twitter.com/ApacheAirflow) on Twitter for the latest news and announcements!
 
+# June 12, 2023
+
+Airflow PMC welcomes new Airflow Committer:
+
+* **Pankaj Singh** ([@o-nikolas](https://github.com/pankajastro)
```
Comment (kaxil) [inline landing-pages/site/content/en/announcements/_index.md:20]:
```suggestion
* **Pankaj Singh** ([@pankajastro](https://github.com/pankajastro)
```

File: landing-pages/site/content/en/announcements/_index.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ menu:
 
 **Note:** Follow [@ApacheAirflow](https://twitter.com/ApacheAirflow) on Twitter for the latest news and announcements!
 
+# June 12, 2023
+
+Airflow PMC welcomes new Airflow Committer:
+
+* **Pankaj Singh** ([@pankajastro](https://github.com/pankajastro)
+
```
Comment (hussein-awala) [inline landing-pages/site/content/en/announcements/_index.md:21]: For the announcement, I am not sure if it should be just on the social media (Linkedin and twitter) or on the site too

**Review by hussein-awala — APPROVED** [review]

File: landing-pages/site/content/en/announcements/_index.md (hunk 2 above)
Comment (kaxil) [reply landing-pages/site/content/en/announcements/_index.md:21]: We always add it to the site

File: landing-pages/site/content/en/announcements/_index.md [hunk 3]
```diff
@@ -13,6 +13,12 @@ menu:
 
 **Note:** Follow [@ApacheAirflow](https://twitter.com/ApacheAirflow) on Twitter for the latest news and announcements!
 
+# June 12, 2023
+
+Airflow PMC welcomes new Airflow Committer:
+
+* **Pankaj Singh** ([@pankajastro](https://github.com/pankajastro)
```
Comment (pankajastro) [inline landing-pages/site/content/en/announcements/_index.md:20]: Look like close `)` is missing.

File: landing-pages/site/content/en/announcements/_index.md (hunk 2 above)
Comment (kaxil) [reply landing-pages/site/content/en/announcements/_index.md:21]: That being said, I think we forgot to add yours @hussein-awala , could you create a PR

File: landing-pages/site/content/en/announcements/_index.md (hunk 3 above)
Comment (kaxil) [reply landing-pages/site/content/en/announcements/_index.md:20]: Yeah , want to create a PR fixing it?

Comment (pankajastro) [reply landing-pages/site/content/en/announcements/_index.md:20]: ok

File: landing-pages/site/content/en/announcements/_index.md (hunk 2 above)
Comment (hussein-awala) [reply landing-pages/site/content/en/announcements/_index.md:21]: Ah okay, I thought something had changed after creating the official page on Linkedin, I will create the PR.

**Pull request merged**